src/main/java/internetshop/controller/AddToBucketController.java

```diff
@@ -21,11 +22,12 @@
     private static BucketService bucketService;
     @Inject
     private static ItemService itemService;
+    private static final Long sesionId = 0L;
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final Long USER_SESSION_ID = 0L;
```